### DISCUSSION TITLE
Who pays for this?

### DISCUSSION THREAD

**Discussion by ulope** [open-meteo/open-meteo#144] [General]
Hi,

this looks like a very well put together project, that makes me wonder who pays for the hosting and development time and what their motivation for doing so and long term plans are?

I can't help but fear that this might be a "bait and switch" type scenario where once enough market saturation has been reached a subscription model is suddenly revealed.

**Comment by patrick-zippenfenig** [comment]
Hi,

currently the project is supported mostly by [GitHub sponsors](https://github.com/sponsors/open-meteo). The operational cost is relatively low and all source data is license and royalty free. Many high-quality commercial weather APIs have large cost associated with licensing fees or running their own weather models.

I am dedicated to keep the API entirely free for non-commercial use. Maybe in the future commercial licences with SLAs follow, but the free non-commercial tier stays the way it is. Can I guarantee this claim? No, but this project is run entirely by myself and not backed by investors. Making the entire code available as open-source hopefully underlines this ambition.

Best, Patrick

**Comment by ulope** [comment]
Awesome, thank you for the detailed response!
